CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 ****
 ## [Unreleased]
 
+## [v4.17.0.1] - 2022-06-21
+### Added
+- Support for APU7
```

Comment:
Link to public description would be grat, but I think there is no any yet. Maybe we could just inform what's the main deal about it (i225 NIC)?

Reply:
Sure, added a short description